### DISCUSSION TITLE
Outages to the Monsoon post-processor and MASS

### DISCUSSION THREAD

**Discussion by mo-mahammedk** [MetOffice/simulation-systems#171] [Monsoon]
There will be two separate pieces of maintenance this week (23rd-24th April). Please see below for details:
 
**Tuesday 23rd April 09:00-11:00 - CHG0111201 – Monsoon Post-processor server (exppostproc01) -**
Monthly security patching of the post-processor server will mean it is unavailable whilst it is rebooted (approximately twenty minutes during the maintenance window).
 
**Wednesday 24th April 10:00-12:00 - CHG0115038 - MASS Planned outage -**
On Wednesday 24th April there will be an outage to MASS in order to apply RHEL security patching on the MASS servers. Between 10:00 and 12:00 all MOOSE 'PUT', 'GET', 'SELECT' and 'FILTER' commands will fail.
 
As always we will post all service status updates here.

**Comment by mo-mahammedk** [comment]
**Tuesday update:** There are no patches to apply to the post-processor server. No reboot needed.

**Reply by mo-roger-milton** [reply]
In addition to MASS patching tomorrow, there will also be a restart of the PBS scheduler on the HPC...

**Restart of PBS server on XCS - Wednesday 24/04/24 - 11:30 BST**

A routine restart of the PBS server on XCS has been scheduled for Wedenesday 24th April at 11:30 local. Running work will not be impacted, but it will not be possible to submit new jobs for 5-10 minutes while the server is in the process of restarting.

**Reply by mo-mahammedk** [reply]
**PBS Update:** PBS server is now back up and running.

**Comment by mo-mahammedk** [comment]
**Wednesday MASS update:** The system is back processing commands.
